config/routes.rb

```diff
@@ -6,5 +6,7 @@
     get  "/about", to: "static_pages#about"
     get  "/contact", to: "static_pages#contact"
     get  "/signup", to: "users#new"
+    post  "/signup", to: "users#create"
+    resources :users
```

Comment:
nếu khai báo 2 line 7 và 8 rồi thì ở line này có cần dùng tới action new và create nữa ko em? dùng cái nào thì chỉ rõ cái đó thôi nha em
```suggestion
    resources :users, only: [ .... ]
```